src/main/java/com/example/tournamentMaker/excel/ExcelUtil.java

```diff
@@ -2,26 +2,44 @@
 
 import lombok.AccessLevel;
 import lombok.NoArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
 import org.apache.poi.ss.usermodel.*;
 
+import java.io.DataOutputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+@Slf4j
 @NoArgsConstructor(access = AccessLevel.PRIVATE)
-class ExcelUtil {
-    static void setThinBorder(CellStyle cellStyle) {
+public class ExcelUtil {
+    public static final int COLUMN_WIDTH_UNIT = 256;
+    public static void setThinBorder(CellStyle cellStyle) {
         cellStyle.setBorderTop(BorderStyle.THIN);
         cellStyle.setBorderBottom(BorderStyle.THIN);
         cellStyle.setBorderLeft(BorderStyle.THIN);
         cellStyle.setBorderRight(BorderStyle.THIN);
     }
-    static void createRows(Sheet sheet, int howMany) {
+    public static void createRows(Sheet sheet, int howMany) {
         for (int i = 0; i < howMany; i++) {
-            sheet.createRow(i);
+            if (sheet.getRow(i) == null) {
+                sheet.createRow(i);
+            }
         }
     }
-    static CellStyle createCellStyle(Workbook workbook, short index) {
+    public static CellStyle createCellStyle(Workbook workbook, short index) {
         CellStyle cellStyle = workbook.createCellStyle();
         cellStyle.setFillForegroundColor(index);
         cellStyle.setFillPattern(FillPatternType.SOLID_FOREGROUND);
+        cellStyle.setAlignment(HorizontalAlignment.CENTER);
         ExcelUtil.setThinBorder(cellStyle);
         return cellStyle;
     }
+    public static boolean createExcelFile(String filePath, Workbook workbook) {
+        try (DataOutputStream dataOutputStream = new DataOutputStream(new FileOutputStream(filePath))) {
+            workbook.write(dataOutputStream);
+            return true;
+        } catch (IOException e) {
+            log.error("An error occurred while writing workbook to file");
```

Comment:
podaj chociaz co to mial byc za workbook , albo co kolwiek wiecej. bo to malo mowi